gemini/node.py

```diff
@@ -50,8 +50,58 @@ def auth_node(node_id, password) -> None:
 
     try:
         response = r.json()
+        
+        filedata = {
+            'node_id': node_id,
+            'password': password,
+        }
+        node_file_writer(filedata)
+
         print(response['token'])
     except ValueError:
-        print("no token returned, credentials do not validate.")
-    
-    
\ No newline at end of file
+        print('no token returned, credentials do not validate.')
+
+
+def node_file_reader() -> dict:
```

Comment:
name this `read_node_file`. and it should return a `Nodefile`. similarly for `write_node_file`, which should accept a Nodefile